kernels/volk/volk_32f_x2_subtract_32f.h

```diff
@@ -200,27 +169,23 @@ static inline void volk_32f_x2_subtract_32f_neon(float* cVector,
                                                  const float* bVector,
                                                  unsigned int num_points)
 {
-    float* cPtr = cVector;
-    const float* aPtr = aVector;
-    const float* bPtr = bVector;
-    unsigned int number = 0;
-    unsigned int quarter_points = num_points / 4;
-
-    float32x4_t a_vec, b_vec, c_vec;
-
-    for (number = 0; number < quarter_points; number++) {
-        a_vec = vld1q_f32(aPtr);
-        b_vec = vld1q_f32(bPtr);
-        c_vec = vsubq_f32(a_vec, b_vec);
-        vst1q_f32(cPtr, c_vec);
-        aPtr += 4;
-        bPtr += 4;
-        cPtr += 4;
-    }
+    const unsigned int quarter_points = num_points / 4;
+
+    for (unsigned int number = 0; number < quarter_points; number++) {
+        float32x4_t a_vec = vld1q_f32(aVector);
+        float32x4_t b_vec = vld1q_f32(bVector);
+
+        float32x4_t c_vec = vsubq_f32(a_vec, b_vec);
 
-    for (number = quarter_points * 4; number < num_points; number++) {
-        *cPtr++ = (*aPtr++) - (*bPtr++);
+        vst1q_f32(cVector, c_vec);
+
+        aVector += 4;
+        bVector += 4;
+        cVector += 4;
     }
+
+    volk_32f_x2_subtract_32f_generic(
+        cVector, aVector, bVector, num_points - quarterPoints * 4);
```

Comment:
```suggestion
        cVector, aVector, bVector, num_points - quarter_points * 4);
```

Reply:
Thanks. I decided to go with `quarterPoints` in this function to make it more consistent with the others. e.g. SSE.